src/EventListener/HookListener.php

```diff
@@ -71,13 +81,29 @@ public function doAddEncore(PageModel $page, LayoutModel $layout, PageRegular $p
         $templateData                     = $layout->row();
 
         // active entries
-        $jsEntries  = [];
-        $cssEntries = [];
+        $jsEntries     = [];
+        $cssEntries    = [];
+        $jsHeadEntries = [];
 
         if (!isset($config['encore']['entries']) || !is_array($config['encore']['entries']) || empty($config['encore']['entries'])) {
             return;
         }
 
+        // add entries from the entrypoints.json
+        if (isset($config['encore']['entrypointsJsons']) && is_array($config['encore']['entrypointsJsons']) && !empty($config['encore']['entrypointsJsons'])) {
```

Comment:
If the encore entries array is empty, you won't reach this section. The check, if the entries array is empty should be after the merge.

Reply:
I switched those completely. This way $config['encore']['entries'] can be null and the entries from  entrypoints.json would still be loaded